go/test/endtoend/preparestmt/main_test.go

```diff
@@ -203,7 +163,8 @@ func TestMain(m *testing.M) {
 		vtgateInstance.ExtraArgs = []string{
 			"--mysql_server_query_timeout", "1s",
 			"--mysql_auth_server_static_file", clusterInstance.TmpDirectory + "/" + mysqlAuthServerStatic,
-			"--mysql_server_version", "8.0.16-7",
+			"--pprof-http",
```

Comment:
How do you get the profiles you've pasted in this PR's description? Do you have to query vtgate's http endpoint during while the test is running?

Reply:
Yes, that is correct.

Reply:
It could be worth adding a comment on how to get the profiles, so we don't have to look into how the next time.

Reply:
Thhe curl commands are pretty standard. I will add them up in the follow up PR.